src/Psalm/Internal/PhpVisitor/Reflector/ClassLikeNodeScanner.php

```diff
@@ -794,30 +793,20 @@ public function finish(PhpParser\Node\Stmt\ClassLike $node): ClassLikeStorage
             }
         }
 
-        $converted_aliases = array_map(
-            function (InlineTypeAlias $t): ?ClassTypeAlias {
-                try {
-                    $union = TypeParser::parseTokens(
-                        $t->replacement_tokens,
-                        null,
-                        [],
-                        $this->type_aliases
-                    );
-
-                    $union->setFromDocblock();
+        $converted_aliases = [];
+        foreach ($this->classlike_type_aliases as $key => $type) {
+            try {
+                $union = TypeParser::parseTokens(
+                    $type->replacement_tokens,
+                    null,
+                    [],
+                    $this->type_aliases
+                );
 
-                    return new ClassTypeAlias(
-                        array_values($union->getAtomicTypes())
-                    );
-                } catch (Exception $e) {
-                    return null;
-                }
-            },
-            $this->classlike_type_aliases
-        );
+                $union->setFromDocblock();
 
-        foreach ($converted_aliases as $key => $type) {
-            if (!$type) {
+                $converted_aliases[$key] = new ClassTypeAlias(array_values($union->getAtomicTypes()));
```

Comment:
`array_filter()` below has became redundant with this change

Reply:
Done.